src/app/GitCommands/Git/GitModule.cs

```diff
@@ -909,9 +909,8 @@ public string GetCommitCountString(ObjectId fromId, string to)
                 return "=";
             }
 
-            return
-                (removed > 0 ? ("-" + removed) : "") +
-                (added > 0 ? ("+" + added) : "");
+            // similar to AheadBehindData
+            return $" (+{added}-{removed})";
```

Comment:
This adds a leading space in dialogs (not requesting a change).

Reply:
Removing the space, keeping the paranthesis.
Screenshot is not updated (hard to see)